stories/components/people/people.properties.js

```diff
@@ -50,3 +50,11 @@ export const PersonCard = () => html`
   <div style="margin-bottom:10px">Person card Click</div>
   <mgt-people show-max="5" person-card="click"></mgt-people>
 `;
+
+export const FallbackDetails = () => html`
+  <mgt-people
+    user-ids="2804bc07-1e1f-4938-9085-ce6d756a32d2, validButNotFound@email.in.ad"
+    fallback-details='[{"id": "validButNotFound@email.in.ad", "mail":"testmail@test.com"},{"mail": "test@test.com"}]'
+  >
+  </mgt-people>
```

Comment:
We shouldn't require id, but we should recommend a displayName is provided so the initials show up. We should also make sure the fallback details are in the right order

```suggestion
  <mgt-people
    user-ids="2804bc07-1e1f-4938-9085-ce6d756a32d2, validButNotFound@email.in.ad"
    fallback-details='[{"mail": "test@test.com"},{"displayName": "validButNotFound@email.in.ad", "mail":"validButNotFound@email.in.ad"}]'
  >
  </mgt-people>
```